templates/ha/hazelcast.xml.erb

```diff
@@ -7,7 +7,7 @@
 -->
 <hazelcast xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd">
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd">
```

Comment:
Bring changes in from:

https://github.com/sonatype/nexus-internal/blob/master/private/assemblies/nexus-pro/src/main/resources/overlay/etc/fabric/hazelcast.xml